### PR TITLE
Upload msgpack-cxx from brew to PyPI

### PR DESCRIPTION
**Pull request by mgovers**
* New package in https://pypi.org/project/msgpack-cxx/
* Successful workflow run in https://github.com/PowerGridModel/msgpack-cxx-pypi/actions/runs/6246880757
  * includes wheel
* Workflow that fails when already exists https://github.com/PowerGridModel/msgpack-cxx-pypi/actions/runs/6247041210

### CODE REVIEW

**Review by TonyXiang8787 — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -7,3 +7,13 @@ This repo is the equivalent of the [Anaconda msgpack-cxx](https://anaconda.org/c
...
 The Python package can be found in [msgpack](https://pypi.org/project/msgpack/).
+
+Credits go to the maintainers of the [msgpack-cxx](https://github.com/msgpack/msgpack-c/tree/cpp_master) package.
+
+## License
+
+The files in this repository are licensed under the Boost Software License - Version 1.0.
+
+The distributed msgpack-cxx package is a redistribution of [msgpack-cxx](https://github.com/msgpack/msgpack-c/tree/cpp_master) and therefore falls under the [msgpack-cxx license model](https://github.com/msgpack/msgpack-c/blob/cpp_master/COPYING) as described in the [COPYING file in the msgpack-cxx](https://github.com/msgpack/msgpack-c/blob/cpp_master/COPYING).
+Since all distributed files are obtained by download in the automation, instead of being explicitly included into this library,
```
Comment (mgovers) [inline README.md]: this is not true anymore due to `src/msgpack_cxx/__init__.py`. can you please revise this block?

File: README.md [hunk 2]
```diff
@@ -7,3 +7,13 @@ This repo is the equivalent of the [Anaconda msgpack-cxx](https://anaconda.org/c
...
+the [msgpack-cxx license](https://github.com/msgpack/msgpack-c/blob/cpp_master/COPYING) is not provided explicitly as a file in this repository either.
```
Comment (mgovers) [inline README.md]: it is now

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+authors = [{ name = "Alliander Dynamic Grid Calculation", email = "dynamic.grid.calculation@alliander.com" }]
+description = "Repackaging of msgpack-cxx distributed via PyPI"
+readme = "README.md"
+license = { text = "BSL-1.0" }
+classifiers = [
+    "Programming Language :: C++",
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    # We only repackage and do not claim ownership over the msgpack-cxx package.
+    # The original license is listed in https://github.com/msgpack/msgpack-c/blob/cpp_master/COPYING
```
Comment (mgovers) [inline pyproject.toml]: should this comment be removed as well?

**Review by TonyXiang8787 — DISMISSED** [review]

**Review by TonyXiang8787 — APPROVED** [review]

**Comment by mgovers** [comment]
Approved :+1:

**Pull request merged**